src/main/java/seedu/address/logic/Messages.java

```diff
@@ -18,8 +18,13 @@ public class Messages {
     public static final String MESSAGE_PERSONS_LISTED_OVERVIEW = "%1$d people listed!";
     public static final String MESSAGE_DUPLICATE_FIELDS =
                 "Multiple values specified for the following single-valued field(s): ";
-    public static final String MESSAGE_INVALID_TELEGRAM = "Cannot find person with telegram handle: %1$s";
+    public static final String MESSAGE_INVALID_TELEGRAM = "Cannot find person with telegram handle: @%1$s";
+    public static final String MESSAGE_TOO_SHORT_TELEGRAM =
```

Comment:
Can change the `MESSAGE_TOO_SHORT_TELEGRAM` message to something like `MESSAGE_LENGTH_INVALID_TELEGRAM` - handle should be 5-32 characters long.

You can also consider using the public static `LENGTH_CONSTRAINTS` string in the Telegram class (once PR #157 is merged in)